alerts/cluster-logging-operator/CollectorHigh403ForbiddenResponseRate.md

```diff
@@ -0,0 +1,125 @@
+# CollectorHigh403ForbiddenResponseRate
+
+## Meaning
+
+The `CollectorHigh403ForbiddenResponseRate` alert indicates that the log collector
+is experiencing a sustained high rate (at least 10% over the last 2 minutes, persisting
+for 5 minutes) of **HTTP 403 Forbidden** responses when attempting to send log
+data to its configured log store.
+
+An **HTTP 403 Forbidden** response indicates that the server understood the request
+but refuses to authorize it.
+
+## Impact
+
+Log collection and forwarding to the destination are impaired,
+resulting in data loss at the configured log store.
+
+## Diagnosis
+
+1. Examine `CollectorHigh403ForbiddenResponseRate` alert details in the OpenShift
+console:
+    - Note the key labels:
+        - `app_kubernetes_io_instance`: The name of the collector.
+        - `namespace`: Namespace of the collector.
+        - `component_id`: The ID of the specific sink in `Vector` that's failing
+        (i.e `output_lokistack_otlp_application`, `output_my_splunk`).
+2. Review the `ClusterLogForwarder` (CLF) configuration:
+    1. Identify the output definition corresponding to the `component_id` obtained
+    from the alert details.
+        - **Non-Lokistack Outputs**: The naming schema is `output_<OUTPUT_NAME>`,
```

Comment:
Probably, this entire explanation about how the "outputs" are called should be better to be in a "common place" and here, only put a link to it?